### PR TITLE
UCC - Evaluate downstream models

### PR DESCRIPTION
**Pull request by yamini**
* Console card + notebook card
* Icon header images
* Sample dataset
* Notebook details markdown file

Note: dataset preview needs to be generated

### CODE REVIEW

**Comment by hhpang** [comment]
lgtm thanks for the copy edits

File: use_cases/gretel.json [hunk 1]
```diff
@@ -71,6 +71,24 @@
         "bytes": 405000
       }
     },
+    {
+      "gtmId": "use-case-downstream-accuracy",
+      "title": "Synthesize data + evaluate ML performance",
```
Comment (hhpang) [inline use_cases/gretel.json:76]: very small thing, can we add "[NEW]" to the card titles?

Comment (yamini) [reply use_cases/gretel.json:76]: new commit

**Review by hhpang — APPROVED** [review]
lgtm! 👾

**Pull request merged**